src/Update/Nix/FetchGit.hs

```diff
@@ -23,66 +21,66 @@ import           Update.Span
 -- Tying it all together
 --------------------------------------------------------------------------------
 
--- | Gather the 'SpanUpdate's for a file at a particular filepath.
+-- | Given the contents of a Nix file, returns the SpanUpdates
+-- all the parts of the file we want to update.
```

Comment:
The comment says that this function is getting the contents of the file, but the type says it's getting the path to the file.